@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -704,7 +704,7 @@ function placePointLabelAtAnchor(
 
     // Glyphs arrangement have been changed remove text buffer object which needs to be
     // re-created.
-    if (forceInvalidation) {
+    if (measureText) {
```

Comment:
Will be more specific, release buffer whenever measurements are taken -> means forced invalidation or bounds where not created -> it ensures us that some cached bounds will not stop us from creating new buffer.